src/components/application_manager/CMakeLists.txt

```diff
@@ -447,6 +454,10 @@ target_link_libraries("AMHMICommandsLibrary" ${LIBRARIES} AMEventEngine)
 add_library("MessageHelper" ${MESSAGE_HELPER_SOURCES})
 target_link_libraries("MessageHelper" ${LIBRARIES})
 
+add_library("SystemTimeLibrary" ${SYSTEM_TIME})
+target_link_libraries("SystemTimeLibrary" ${LIBRARIES} AMEventEngine)
+add_dependencies("MessageHelper" HMI_API MOBILE_API)
```

Comment:
@AByzhynar Is it related to system time? If not, please move this line closer to adding MessageHelper library